src/Sarif/IResultLogWriter.cs

```diff
@@ -17,7 +33,15 @@ public interface IResultLogWriter
         /// <exception cref="ArgumentNullException">Thrown if <paramref name="info"/> is null.</exception>
         /// <param name="toolInfo">The tool information to write.</param>
         /// <param name="runInfo">The run information to write.</param>
-        void WriteToolAndRunInfo(ToolInfo toolInfo, RunInfo runInfo);
+        void WriteRunInfo(RunInfo runInfo);
+
```

Comment:
`<paramref name="runInfo" /> on Line 33.

`toolInfo` is not a parameter of this method.